service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/C21OrderControllerTest.java

```diff
@@ -137,4 +176,58 @@ private AboutToStartOrSubmitCallbackResponse makeRequest(CallbackRequest request
         return mapper.readValue(mvc.getResponse()
             .getContentAsByteArray(), AboutToStartOrSubmitCallbackResponse.class);
     }
+
+    private void makeSubmittedRequest(CallbackRequest request)
+        throws Exception {
+        mockMvc
+            .perform(post("/callback/create-order/submitted")
+                .header("authorization", AUTH_TOKEN)
+                .header("user-id", USER_ID)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(request)))
+            .andExpect(status().isOk())
+            .andReturn();
+    }
+
+    private CallbackRequest buildCallbackRequest() {
+        return CallbackRequest.builder()
+            .caseDetails(CaseDetails.builder()
+                .id(19898989L)
+                .data(ImmutableMap.of(
+                    "c21Orders", createC21Orders(),
+                    "hearingDetails", createHearingBookings(dateIn3Months),
+                    "respondents1", createRespondents(),
+                    "caseLocalAuthority", LOCAL_AUTHORITY_CODE,
+                    "familyManCaseNumber", FAMILY_MAN_CASE_NUMBER))
+                .build())
+            .build();
+    }
+
+    private Map<String, Object> similarParameters() {
```

Comment:
Not sure if I like the name similar, similar to what? commonNotification?

Reply:
updated